src/main/java/Duke.java

```diff
@@ -1,10 +1,57 @@
+import java.util.Scanner;
+
 public class Duke {
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
+        String logo = "     ____        _        \n"
+                + "    |  _ \\ _   _| | _____ \n"
+                + "    | | | | | | | |/ / _ \\\n"
+                + "    | |_| | |_| |   <  __/\n"
+                + "    |____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+
+        String greet = "    ____________________________________________________________\n"
+                + "     Hello! I'm Duke \n"
+                + "     What can I do for you? \n"
+                + "    ____________________________________________________________\n";
+        System.out.println(greet);
+
+        String line;
+        final int TOTAL_TASKS = 100; // Total number of tasks
```

Comment:
I like how you set the number of tasks as a constant variable so it can be set in a single place, instead of just using 100 everywhere.